frontend/src/metabase/common/components/EntityPicker/components/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./EntityPickerModal";
+export * from "./EntityPickerSearch";
+export * from "./LoadingSpinner";
+export * from "./NestedItemPicker";
+export * from "./CollectionPicker";
```

Comment:
I think we should only export
- `NestedItemPicker`
- `ItemPicker`
- `CollectionPicker`

I don't think the others are useful outside this directory